react/class_components/class_based_components.md

```diff
@@ -131,14 +132,14 @@ class ClassInput extends Component {
     return (
       <section>
         <h3>{this.props.name}</h3>
-        {/* The input field to enter To-Do's */}
+        {/* The input field to enter To-Dos */}
```

Comment:
Nit: Just to stay consitent with the code and the lesson text, can we change this and the other comment to "todos" please?

 ```suggestion
        {/* The input field to enter Todos */}
```